Readme instructs to use non-existent make command

The readme file says to run

    make dev

However, there is no make target called `dev` in the makefile.

Btw. I think you should change the other make lines too bye prepending `sudo` as it currently is not evident that those commands need to be run as root.

@pik694 I get this error: "ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/lib/python3.7/site-packages/plasma-contracts.egg-link'
Consider using the `--user` option or check the permissions."

Command and complete output:

```
$ make init
npm install --python=python2
npm WARN ts-node@8.4.1 requires a peer of typescript@>=2.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/ganache-core-sc/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 323078 packages in 8.106s
found 10 vulnerabilities (6 low, 1 moderate, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details
make -C python_tests/   init dev
make[1]: Entering directory '/tmp/tmp.Go9rVCQwcb/plasma-contracts/plasma_framework/python_tests'
(test -d openzeppelin-solidity || git clone https://github.com/OpenZeppelin/openzeppelin-solidity.git) && cd openzeppelin-solidity && git checkout v2.3.0
Cloning into 'openzeppelin-solidity'...
remote: Enumerating objects: 91, done.
remote: Counting objects: 100% (91/91), done.
remote: Compressing objects: 100% (82/82), done.
remote: Total 15101 (delta 17), reused 42 (delta 7), pack-reused 15010
Receiving objects: 100% (15101/15101), 6.24 MiB | 4.03 MiB/s, done.
Resolving deltas: 100% (10271/10271), done.
Note: switching to 'v2.3.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 132e442 2.3.0
pip install -e .
Obtaining file:///tmp/tmp.Go9rVCQwcb/plasma-contracts/plasma_framework/python_tests
Requirement already satisfied: rlp==1.1.0 in /usr/lib/python3.7/site-packages (from plasma-contracts==0.0.1) (1.1.0)
Requirement already satisfied: py-solc-simple@ git+https://github.com/omisego/py-solc-simple@plasma_contracts_tmp_compilation from git+https://github.com/omisego/py-solc-simple@plasma_contracts_tmp_compilation in /usr/lib/python3.7/site-packages (from plasma-contracts==0.0.1) (0.0.14)
Requirement already satisfied: web3==5.0.0 in /usr/lib/python3.7/site-packages (from plasma-contracts==0.0.1) (5.0.0)
Requirement already satisfied: eip712-structs==1.1.0 in /usr/lib/python3.7/site-packages (from plasma-contracts==0.0.1) (1.1.0)
Requirement already satisfied: eth_tester==0.2.0b2 in /usr/lib/python3.7/site-packages (from plasma-contracts==0.0.1) (0.2.0b2)
Requirement already satisfied: eth-utils<2,>=1.0.2 in /usr/lib/python3.7/site-packages (from rlp==1.1.0->plasma-contracts==0.0.1) (1.8.1)
Requirement already satisfied: py-solc-x==0.4.0 in /usr/lib/python3.7/site-packages (from py-solc-simple@ git+https://github.com/omisego/py-solc-simple@plasma_contracts_tmp_compilation->plasma-contracts==0.0.1) (0.4.0)
Requirement already satisfied: jsonschema<3,>=2.6.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (2.6.0)
Requirement already satisfied: eth-abi<3.0.0,>=2.0.0b6 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (2.0.0)
Requirement already satisfied: protobuf<4,>=3.0.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (3.11.0)
Requirement already satisfied: requests<3.0.0,>=2.16.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (2.22.0)
Requirement already satisfied: hexbytes<1.0.0,>=0.1.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (0.2.0)
Requirement already satisfied: eth-account<0.5.0,>=0.4.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (0.4.0)
Requirement already satisfied: eth-typing<3.0.0,>=2.0.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (2.2.1)
Requirement already satisfied: lru-dict<2.0.0,>=1.1.6 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (1.1.6)
Requirement already satisfied: ipfshttpclient<1,>=0.4.12 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (0.4.12)
Requirement already satisfied: websockets<8.0.0,>=7.0.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (7.0)
Requirement already satisfied: eth-hash[pycryptodome]<1.0.0,>=0.2.0 in /usr/lib/python3.7/site-packages (from web3==5.0.0->plasma-contracts==0.0.1) (0.2.0)
Requirement already satisfied: pysha3>=1.0.2 in /usr/lib/python3.7/site-packages (from eip712-structs==1.1.0->plasma-contracts==0.0.1) (1.0.2)
Requirement already satisfied: semantic-version<3.0.0,>=2.6.0 in /usr/lib/python3.7/site-packages (from eth_tester==0.2.0b2->plasma-contracts==0.0.1) (2.8.3)
Requirement already satisfied: eth-keys<0.3.0,>=0.2.1 in /usr/lib/python3.7/site-packages (from eth_tester==0.2.0b2->plasma-contracts==0.0.1) (0.2.4)
Requirement already satisfied: cytoolz<1.0.0,>=0.10.1 in /usr/lib/python3.7/site-packages (from eth-utils<2,>=1.0.2->rlp==1.1.0->plasma-contracts==0.0.1) (0.10.1)
Requirement already satisfied: toolz<1,>0.8.2 in /usr/lib/python3.7/site-packages (from eth-utils<2,>=1.0.2->rlp==1.1.0->plasma-contracts==0.0.1) (0.10.0)
Requirement already satisfied: parsimonious<0.9.0,>=0.8.0 in /usr/lib/python3.7/site-packages (from eth-abi<3.0.0,>=2.0.0b6->web3==5.0.0->plasma-contracts==0.0.1) (0.8.1)
Requirement already satisfied: six>=1.9 in /usr/lib/python3.7/site-packages (from protobuf<4,>=3.0.0->web3==5.0.0->plasma-contracts==0.0.1) (1.12.0)
Requirement already satisfied: setuptools in /usr/lib/python3.7/site-packages (from protobuf<4,>=3.0.0->web3==5.0.0->plasma-contracts==0.0.1) (41.2.0)
Requirement already satisfied: chardet>=3.0.2 in /usr/lib/python3.7/site-packages (from requests<3.0.0,>=2.16.0->web3==5.0.0->plasma-contracts==0.0.1) (3.0.4)
Requirement already satisfied: idna>=2.5 in /usr/lib/python3.7/site-packages (from requests<3.0.0,>=2.16.0->web3==5.0.0->plasma-contracts==0.0.1) (2.8)
Requirement already satisfied: urllib3>=1.21.1 in /usr/lib/python3.7/site-packages (from requests<3.0.0,>=2.16.0->web3==5.0.0->plasma-contracts==0.0.1) (1.25.6)
Requirement already satisfied: eth-rlp<1,>=0.1.2 in /usr/lib/python3.7/site-packages (from eth-account<0.5.0,>=0.4.0->web3==5.0.0->plasma-contracts==0.0.1) (0.1.2)
Requirement already satisfied: attrdict<3,>=2.0.0 in /usr/lib/python3.7/site-packages (from eth-account<0.5.0,>=0.4.0->web3==5.0.0->plasma-contracts==0.0.1) (2.0.1)
Requirement already satisfied: eth-keyfile<0.6.0,>=0.5.0 in /usr/lib/python3.7/site-packages (from eth-account<0.5.0,>=0.4.0->web3==5.0.0->plasma-contracts==0.0.1) (0.5.1)
Requirement already satisfied: multiaddr>=0.0.7 in /usr/lib/python3.7/site-packages (from ipfshttpclient<1,>=0.4.12->web3==5.0.0->plasma-contracts==0.0.1) (0.0.8)
Requirement already satisfied: pycryptodome<4,>=3.6.6; extra == "pycryptodome" in /usr/lib/python3.7/site-packages (from eth-hash[pycryptodome]<1.0.0,>=0.2.0->web3==5.0.0->plasma-contracts==0.0.1) (3.9.4)
Requirement already satisfied: base58 in /usr/lib/python3.7/site-packages (from multiaddr>=0.0.7->ipfshttpclient<1,>=0.4.12->web3==5.0.0->plasma-contracts==0.0.1) (1.0.3)
Requirement already satisfied: netaddr in /usr/lib/python3.7/site-packages (from multiaddr>=0.0.7->ipfshttpclient<1,>=0.4.12->web3==5.0.0->plasma-contracts==0.0.1) (0.7.19)
Requirement already satisfied: varint in /usr/lib/python3.7/site-packages (from multiaddr>=0.0.7->ipfshttpclient<1,>=0.4.12->web3==5.0.0->plasma-contracts==0.0.1) (1.0.2)
Installing collected packages: plasma-contracts
  Found existing installation: plasma-contracts 0.0.1
    Uninstalling plasma-contracts-0.0.1:
ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/lib/python3.7/site-packages/plasma-contracts.egg-link'
Consider using the `--user` option or check the permissions.

make[1]: *** [Makefile:3: init] Error 1
make[1]: Leaving directory '/tmp/tmp.Go9rVCQwcb/plasma-contracts/plasma_framework/python_tests'
make: *** [Makefile:18: init_python] Error 2
```

I just ran this on a fresh and up-to-date clone of this repository.

--- a/README.md
+++ b/README.md
@@ -77,6 +77,7 @@ Run truffle, passing in the network e.g.
 
 
 ### Building and running the python tests
+All the code is in the `plasma_framework/python_tests` directory, so go there first.
 
 Installing dependencies needed for compilation:
 ```

--- a/README.md
+++ b/README.md
@@ -77,6 +77,7 @@ Run truffle, passing in the network e.g.
 
 
 ### Building and running the python tests
+We suggest running the following commands with an active python virtual environment ex. `venv`.
 All the code is in the `plasma_framework/python_tests` directory, so go there first.
 
 Installing dependencies needed for compilation:

